### PR TITLE
docs/tutorial: Better images and check for §1.1.4

### PR DESCRIPTION
**Pull request by 0cjs**
The Waterfall image page was incorrect; at this point in the tutorial
the actual system displays an empty page instead of the "runtests"
waterfall line.

Since the point is to confirm not only that the master is running, but
also that the worker we started is running and has connected to the
master, we instead display the Builders page which should display the
connected worker/runner.

We also replace the top page (index) image with a screenshot from the
same run so that the Buildbot's title in the image now matches what's
in the configuration file and in the second iamge.

## Contributor Checklist:

* [N/A] I have updated the unit tests
* [NO] I have created a file in the `master/buildbot/newsfragment` directory (and read the `README.txt` in that directory)
* [ YES] I have updated the appropriate documentation

### CODE REVIEW

**Comment by 0cjs** [comment]
## Developer Notes

* I don't think this is a big enough change to warrant a newsfragment, though I'll make one if others think it's useful.
* I don't know off-hand how to build the docs and view them locally, so if someone can save the time of researching this by checking it for me or telling me how to do this, that would be very helpful.

File: master/docs/tutorial/docker.rst [hunk 1]
```diff
@@ -69,10 +69,10 @@ You should now be able to go to http://localhost:8010 and see a web page similar
 .. image:: _images/index.png
    :alt: index page
 
-Click on the `Waterfall Display link <http://localhost:8010/#/waterfall>`_ and you get this:
+Click on "Builds" at the left to open the submenu and then `Builders <http://localhost:8010/#/builders>`_ to see that the worker you just started has connected to the master:
```
Comment (tardyp) [inline master/docs/tutorial/docker.rst]: sub-menu

File: master/docs/tutorial/firstrun.rst [hunk 2]
```diff
@@ -187,10 +187,10 @@ You should now be able to go to http://localhost:8010, where you will see a web
 .. image:: _images/index.png
    :alt: index page
 
-Click on the `Waterfall Display link <http://localhost:8010/#/waterfall>`_ and you get this:
+Click on "Builds" at the left to open the submenu and then `Builders <http://localhost:8010/#/builders>`_ to see that the worker you just started has connected to the master:
```
Comment (tardyp) [inline master/docs/tutorial/firstrun.rst]: sub-menu

File: master/docs/tutorial/docker.rst (hunk 1 above)
Comment (0cjs) [reply master/docs/tutorial/docker.rst]: I see that sort of incorrect suggestion a lot from spell checkers, for some reason. But according to the [OED](https://en.oxforddictionaries.com/definition/submenu) and [Merriam-Webster](https://www.merriam-webster.com/dictionary/submenu) it's fine (with only MW listing "sub-menu" as a variant).

I prefer it without the hyphen, parallel to some other common words taking a "sub" prefix ("substation" over "sub-station", "substandard" over "sub-standard") but I'm not going to bikeshed, so take this fix if you feel strongly about it.

```suggestion
Click on "Builds" at the left to open the sub-menu and then `Builders <http://localhost:8010/#/builders>`_ to see that the worker you just started has connected to the master:
```

File: master/docs/tutorial/firstrun.rst (hunk 2 above)
Comment (0cjs) [reply master/docs/tutorial/firstrun.rst]: As above.

```suggestion
Click on "Builds" at the left to open the sub-menu and then `Builders <http://localhost:8010/#/builders>`_ to see that the worker you just started has connected to the master:
```

**Comment by seankelly** [comment]
The word list is at `master/docs/spelling_wordlist.txt`.

**Comment by tardyp** [comment]
@0cjs I have no strong opinion, please update the word list for spell checker or change the spelling of your contribution. Thank you very much!

**Comment by 0cjs** [comment]
@tardyp I'm sorry about the harshness of my previous response; I incorrectly took your comment as a criticism of my use of English rather than just a request to update the spelling dictionary. That turns out to be a simple fix and, now that I know how that test works, I think it's a worthwhile test to keep.

I've pushed up a new version with the spelling dictionary updated, so I think this change is now fine. However, errors are appearing in the build that appear to be unrelated and from other commits recently added to `master`. Once those are fixed on `master` this commit can be rebased on to those and should be fine. You can do that if you like (I've invited you access to my fork of the repo, in case that's necessary) or I will do it when I notice new changes on `master`.

**Comment by 0cjs** [comment]
I've tacked on another commit with a minor documentation fix, since it didn't seem worth a separate PR. But I'll do a separate one if there's need for it.

**Pull request merged**